source/grammar/qasm3.g4

```diff
@@ -276,34 +279,73 @@ logicalOperator
     | '||'
     ;
 
+shiftOperator
```

Comment:
Don't think this is necessary. It will just make the resulting parse tree a longer hierarchy. I think the way it was before was clear.